bgpd/bgp_filter.c

```diff
@@ -391,11 +478,13 @@ bool config_bgp_aspath_validate(const char *regstr)
 }
 
 DEFUN(as_path, bgp_as_path_cmd,
-      "bgp as-path access-list WORD <deny|permit> LINE...",
+      "bgp as-path access-list WORD [seq (1-4294967295)] <deny|permit> LINE...",
```

Comment:
We already allow `seq 0` for community-lists. Maybe it's worth doing here the same?